lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -308,6 +316,25 @@
   stampTemplate('unoptimized-images-tmpl', document.body);
 }
 
+function deprecationsTest() {
+  stampTemplate('deprecations-tmpl', document.head);
+
+  // TODO: some deprecated API messages are not currently being logged correctly.
+  // See: https://crbug.com/680832
+  // try {
+  //   MediaStreamTrack.getSources(); // FAIL
+  // } catch(e) {}
+
+  // console.timeline(); // FAIL
+
+  // FAIL
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', 'dbw_tester.html', false);
```

Comment:
sorry, forgot to include earlier: on the topic of cross origin requests, can you change the `https://unknown.com` (which appears to be a real site, just not https) to something like `https://unknown.invalid`, which is reserved for testing?